### DISCUSSION TITLE
Test scenario with only one vuser?

### DISCUSSION THREAD

**Discussion by mnylen** [artilleryio/artillery#2706] [Artillery]
Hey,

when developing a scenario, I'd find it quite useful to be able to run the scenario only once with a single vuser and then stop the run, instead of doing i.e. one vuser per second.

Is there a way to do this with load phases, or is there some kind of option to just run the scenario once?

**Comment by hassy** [comment]
hi @mnylen 👋 `artillery run --solo` is what you need! which is not documented right now - will fix that. :)

**Reply by mnylen** [reply]
Thank you! That seems to do the trick indeed.

**Comment by skerit** [comment]
When I run it with `--solo` nothing happens, not a single user is used.
Is this argument supported by now, or not?
